src/js/view/controls/components/thumbnails.mixin.js

```diff
@@ -82,7 +83,8 @@ const ThumbnailsMixin = {
     },
 
     showThumbnail: function(seconds) {
-        if (this.thumbnails.length < 1) {
+        // Don't attempt to set thumbnail for small players or when a thumbnail doesn't exist
+        if (this._model.get('containerWidth') <= 420 || this.thumbnails.length < 1) {
```

Comment:
This only applies when the player is set up with a width smaller than 420, not if it's resized to a smaller breakpoint

Reply:
The model gets the updated player width when the player gets resized. The squash breakpoint toggles aren't working correctly atm.

Reply:
What if the thumbnail is open while the player resizes?

Reply:
Highly unlikely that the user is hovering over the timeslider and resizing the player at the same time. Even then, there'll be eventual consistency once the model gets updated.